docs/stdlib/array.rst

```diff
@@ -141,14 +140,15 @@ Reference
         db> select [(x := 1, y := 1), (x := 2, y := 3.3)][1];
         {(x := 2, y := 3.3)}
 
-    Negative indexing is supported:
+    This operator may also be used when trying to negatively index elements.
 
     .. code-block:: edgeql-repl
 
         db> select [1, 2, 3][-1];
         {3}
 
-    Referencing a non-existent array element will result in an error:
+    However, referencing an non-existent element of an array will result in
```

Comment:
```suggestion
    However, referencing a non-existent element of an array will result in
```
?

Reply:
I'm also not sure why "However" is used here; non-existing element error is really not connected in any way to the negative indexing.